openapi-spec/swagger.yaml

```diff
@@ -335,7 +397,7 @@ paths:
               schema:
                 type: array
                 items:
-                  $ref: '#/components/schemas/ProfileSpec'
+                  $ref: '#/components/schemas/ProfileRespSpec'
               example:
               - id: 5d8c3732-a248-50ed-bebc-539a6ffd25c1
```

Comment:
does the example match the ProfileRespSpec ?

Reply:
Yes. Please see the line no: 4027